src/plugin.js

```diff
@@ -742,7 +752,7 @@ const contribAdsPlugin = function(options) {
     return !player.ads.shouldPlayContentBehindAd(player) &&
             player.ads.isAdPlaying() &&
             player.tech_.featuresNativeTextTracks &&
-            videojs.browser.IS_IOS &&
+            (videojs.browser.IS_IOS || DESKTOP_SAFARI_VERSION === 11) &&
```

Comment:
How about `DESKTOP_SAFARI_VERSION >= 11` instead?

Reply:
Yeah, that is a safer bet. I wrote it this way since we technically don't know if this fix will be needed in future versions of Safari, but including Safari 12+ does have the potential to save us some headaches down the road, and doesn't have a downside I can think of. I'll make the change.